app/helpers/uploads_helper.rb

```diff
@@ -0,0 +1,69 @@
+module UploadsHelper
+  def uploads_meta_tags
+    [
+      tag(:meta, name: :upload_bucket_name, content: ENV["UPLOAD_BUCKET_NAME"]),
+      tag(:meta, name: :upload_bucket_prefix, content: uploads_prefix),
+      tag(:meta, name: :upload_s3_endpoint_host, content: ENV["UPLOAD_S3_ENDPOINT_HOST"]),
+      tag(:meta, name: :upload_signing_service_key_id, content: uploads_key_id),
+      tag(:meta, name: :upload_signing_service_url, content: uploads_service_url)
+    ].join("\n    ").html_safe
+  end
```

Comment:
What are the `meta_tags`? Is this purely an AWS thing?

Reply:
These insert tags into the HTML `<head>`, from the Application layout.  The page source will have a bunch of things like:

```html
<meta name="upload_bucket_prefix" content="uploads/feeder" />
```

Honestly, I can't remember why I did that, instead of using [Stimulus values](https://stimulus.hotwired.dev/reference/values) to pass these to the controller.

Maybe just trying to save some characters, vs typing out a bunch of `data_upload_bucket_prefix_value=`?  Whatever past-ryan thought, i'm not feeling opinionated on it.  Would it be clearer if I moved them?

Reply:
Also not too opinionated. It's understandable as it is and why it lives here.